src/Orleans/LogConsistency/IProtocolServices.cs

```diff
@@ -0,0 +1,141 @@
+﻿
+using System;
+using System.Net;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Orleans.Providers;
+using Orleans.Runtime;
+using Orleans.MultiCluster;
+using Orleans.GrainDirectory;
+
+namespace Orleans.LogConsistency
+{
+    /// <summary>
+    /// Functionality for use by log view adaptors that use custom consistency or replication protocols.
+    /// Abstracts communication between replicas of the log-consistent grain in different clusters.
+    /// </summary>
+    public interface IProtocolServices
```

Comment:
Yes, that makes sense. I will go with ILogConsistencyProtocolServices, since it makes it very clear that this is related to the LogConsistencyProvider and the LogConsistentGrain classes.